lib/encumber.rb

```diff
@@ -374,42 +377,74 @@ def move_mouse_to_point(x, y)
 
     end
 
-    def simulate_left_click(xpath, flags=[])
+    def location_for_simulate_mouse_event(original_location, current_location, destination_location, i)
```

Comment:
You should split this function in two parts. It's weird to ask for a location and in this method a `simulateMouseMovedOnPoint` is generated.